HolyshiftService/Auth/JwtService.cs

```diff
@@ -0,0 +1,76 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using HolyShift.Config;
+using HolyShift.Database.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+
+namespace HolyShift.Auth;
+
+public class JwtService
```

Comment:
Create an interface for this class and use that interface instead (for testability)